augur/merge.py

```diff
@@ -96,7 +104,7 @@ def run(args):
         raise AugurError(dedent(f"""\
             All metadata inputs must be assigned a name, e.g. with NAME=FILE.
 
-            The following inputs were missing a name:
+            The following {_n("input was", "inputs were", len(unnamed))} missing a name:
```

Comment:
Nice, I didn't know about `ngettext`. If/when this goes through I'd like to replace 37932e1d51604b21bfed39890757f4cde2ee490d.

Reply:
Nod. I also think your solution in that commit is just fine. I started with something similar but it got uglier than I wanted, so I switched to `ngettext`.

Reply:
https://github.com/nextstrain/augur/pull/1596